contracts/BatchExchangeViewer.sol

```diff
@@ -16,6 +18,41 @@ contract BatchExchangeViewer {
         batchExchange = exchange;
     }
 
+    struct Balance {
+        address token; // The token for which this balance is reported
+        uint256 availableSellAmount; // Amount available to trade in the auction for which it was requested
+        uint256 pendingDeposit; // Amount that was deposited in the requested auction. Will be tradeable in the next auction
+        uint256 withdrawableBalance; // Amount that can be withdrawn in the requested auction
+    }
+
+    /** @dev Returns the user's token balances for the auction that is currently being settled
```

Comment:
Previously we used "auction that is still accepting orders" and "auction that is currently being solved". Is "being settled" one of those or a new meaning?

Reply:
Good point. It's the same. Will unify the wording

Reply:
I’m Not sure what balances have to do with auction batch state. Is there some more context here that I am missing?

Reply:
Depending on the auction index your balance can differ (as pending deposits/withdraws might be applied)

Reply:
Ya surely, but why are we talking about whether or not the "auction that is still accepting orders" or other things regarding the auction here?

Reply:
Note, that you are not providing an auction Index into this method. Therefore, one thing a user might ask themselves is which balances are you going to return to me? 
The ones that are valid/available in the auction that is currently being solved, or the ones that will be valid only in the next auction.

Due to pending deposits/withdraws those amounts can differ.